src/main/python/envbuilder/cli.py

```diff
@@ -0,0 +1,276 @@
+# Copyright (c) 2020 Intel Corporation
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#      http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import logging
+import os
+import re
+
+from colors import color
+
+from envbuilder import API
+from envbuilder import DockerImageSearch
+
+logger = logging.getLogger(__name__)
+
+class EnvBuilder():
+    def __init__(self, args):
+        self.args = args
+        self.image_search = DockerImageSearch()
+        self.api = self.get_client()
+
+        self.envfile = EnvBuilder.read_env_file(args.envfile)
+        self.compose_env_vars = EnvBuilder.env_to_dict(self.envfile)
+        self.docker_repository = 'edgexfoundry'
+
+        self.repo_map = {
+            'edgex-go': 'CORE_EDGEX_VERSION',
+            'app-service-configurable': 'APP_SERVICE_VERSION',
+            'edgex-ui-go': 'EDGEX_UI_VERSION',
+            'device-bacnet-c': 'DEVICE_BACNET_VERSION',
+            'device-camera-go': 'DEVICE_CAMERA_VERSION',
+            'device-grove-c': 'DEVICE_GROVE_VERSION',
+            'device-modbus-go': 'DEVICE_MODBUS_VERSION',
+            'device-mqtt-go': 'DEVICE_MQTT_VERSION',
+            'device-random': 'DEVICE_RANDOM_VERSION',
+            'device-rest-go': 'DEVICE_REST_VERSION',
+            'device-snmp-go': 'DEVICE_SNMP_VERSION',
+            'device-virtual-go': 'DEVICE_VIRTUAL_VERSION',
+        }
+
+    def process_env_file(self):
+        if self.args.no_lookup is False:
```

Comment:
from PEP8: "Don't compare boolean values to True or False using == (or is)"
https://www.python.org/dev/peps/pep-0008/#programming-recommendations